package.json

```diff
@@ -29,14 +29,18 @@
     "eslint": "^9.8.0",
     "gherkin-lint": "^4.2.2",
     "rollup": "^4.19.2",
-    "tailwindcss": "^3.3.5"
+    "tailwindcss": "^3.3.5",
+    "vitepress": "^1.3.1"
```

Comment:
`vitepress` 1.4.0 is out. Does it make sense to bump before merge it the PR?

Reply:
The installed version is 1.4.0 if you check the lock file but for some reason the package.json doesn't change when updating. I'm not sure what command does it. I updated package.json and reran `yarn install` manually.